.gitignore

```diff
@@ -137,3 +137,7 @@ DO_NOT_COMMIT
 
 # Hive Metastore from Spark
 metastore_db
+
+# dependency managment
+Pipfile
+Pipfile.lock
```

Comment:
Just learned more about this [here](https://stackoverflow.com/questions/46330327/how-are-pipfile-and-pipfile-lock-used). Love the change